src/components/LayoutComponents/LayoutNavbar.tsx

```diff
@@ -13,22 +13,26 @@ const LayoutNavbar: FC<LayoutNavbarProps> = ({ navOpened, setNavOpened }) => {
         {
             label: 'Home',
             targetUrl: '/',
-            icon: <IconHome size='24px' />
+            icon: <IconHome size='24px' />,
+            setNavOpened: setNavOpened
```

Comment:
remove the setNavOpened value from the array. The variable has already been passed from the parent into the whole child component. Adding the variable here does nothing

Reply:
`setNavOpened: setNavOpened` should be removed from the array